### PR TITLE
ODS-5475 Move ODS/API: Publish to Azure artifacts from Github Actions

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
I will delete teamcity build https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_OdsApiPublishToAzure?branch=&mode=builds#all-projects once this PR is approved

**Review by bptillman — COMMENTED** [review]
These changes all look good. Just one comment to get your thoughts about the runner we are using and otherwise this is good.

File: build.github.ps1 [hunk 1]
```diff
@@ -48,7 +48,7 @@ param(
     
     [String] $WebApiId  = 'EdFi.Suite3.Ods.WebApi',
     
-    [String] $DatabasesId = 'EdFi.Suite3.Ods.RestApi.Databases'
+    [String] $DatabasesId = 'EdFi.Suite3.RestApi.Databases'
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply build.github.ps1:51]: Thanks

File: .github/workflows/Publish to Azure Artifacts manual.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+env:
+  AZURE_ARTIFACT_URL: "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json"
+  AZURE_ARTIFACT_NUGET_KEY: ${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}
+  VSS_NUGET_EXTERNAL_FEED_ENDPOINTS : '{"endpointCredentials": [{"endpoint": "https://pkgs.dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_packaging/EdFi/nuget/v3/index.json","password": "${{ secrets.AZURE_ARTIFACTS_PERSONAL_ACCESS_TOKEN }}"}]}'
+  CURRENT_BRANCH: ${{ GITHUB.REF_NAME }}
+
+jobs:
+  build:
+
+    runs-on: windows-latest
```
[the remark this answers is not in the document]
Comment (semalaiappan) [reply .github/workflows/Publish to Azure Artifacts manual.yml]: Yes . Updated

**Review by bptillman — APPROVED** [review]

**Comment by semalaiappan** [comment]
Teamcity build is removed now https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_OdsApiPublishToAzure?branch=&mode=builds#all-projects

**Pull request merged**